template/web/package.json

```diff
@@ -27,7 +27,7 @@
     "@radix-ui/react-navigation-menu": "^1.1.4",
```

Comment:
If we are not use Privy anymore, let's remove it from here.

Reply:
Removed privy dependency